src/Microsoft.Health.SqlServer/Features/Schema/Migrations/BaseSchema.sql

```diff
@@ -4,10 +4,10 @@
 /*************************************************************
     Configure database
 **************************************************************/
-
 -- Enable RCSI
+-- ROLLBACK IMMEDIATE will cut off live connections and roll back any open transactions immediately
 IF ((SELECT is_read_committed_snapshot_on FROM sys.databases WHERE database_id = DB_ID()) = 0) BEGIN
-    ALTER DATABASE CURRENT SET READ_COMMITTED_SNAPSHOT ON
+    ALTER DATABASE CURRENT SET READ_COMMITTED_SNAPSHOT ON WITH ROLLBACK IMMEDIATE
```

Comment:
Since this will be executed only once when the schema is null, so I think there will not be any transactions running at this point and its safe to ensure that any connection by server is closed.

Reply:
Tested this locally with the test web project and I'm no longer seeing any delays in the base schema execution, and the `apply --next` command succeeds 👍.

I did a bit of reading about `WITH ROLLBACK IMMEDIATE` and found: "this option will rollback all uncommitted transactions and reset all connections" (from [stackoverflow](https://stackoverflow.com/questions/34589660/sql-server-alter-database-with-rollback-immediate)). Did the setup for SQL MI create a connection that was blocking the base schema from being excuted? Just trying to figure out why there is now an open connection that we need to reset.

Reply:
If someone were running an older version of the code with schema version 1 or 2, and upgraded the FHIR service directly to the latest and ran schema migration, would this script get run?

Reply:
@YazanMSFT  baseSchema.sql would be executed but not alter database scripts since if condition would not evaluate to true.

Reply:
Thanks, that makes sense. We do have the option of specifying WITH ROLLBACK AFTER [n] rather than immediate, to give a chance for stuff to complete before forcing them to rollback. But if we're sure it won't get run then immediate is fine.

Reply:
Good point. I initially gave it a thought before creating this PR to ROLLBACK AFTER 30 seconds or so. But then realizing that since schema would be null at this point so no transactions should ideally run.

Reply:
@rotodd Good question. I took a look at this [PR](https://github.com/microsoft/healthcare-shared-components/pull/102) and seems connection is scoped to using block so not sure what's causing the open connection. But I think IMMEDIATE ROLLBACK is okay to put in since no schema would exist at this point(and more comments above)